src/modules/suggested-fees/service.ts

```diff
@@ -0,0 +1,38 @@
+import { HttpService } from "@nestjs/axios";
+import { Injectable } from "@nestjs/common";
+
+import { AppConfig } from "../configuration/configuration.service";
+
+type SuggestedFeesApiParams = {
+  amount: string;
+  token: string;
+  destinationChainId: number;
+  originChainId: number;
+};
+
+type SuggestedFeesApiResponse = {
+  data: {
+    capitalFeePct: string;
+    capitalFeeTotal: string;
+    relayGasFeePct: string;
+    relayGasFeeTotal: string;
+    relayFeePct: string;
+    relayFeeTotal: string;
+    lpFeePct: string;
+    timestamp: string;
+    isAmountTooLow: boolean;
+  };
+};
+
+@Injectable()
+export class SuggestedFeesService {
```

Comment:
I don't think we need a separate module for this service. Actually you thought the same since you added this service to the providers list of the `ScraperModule`. We can move this file to `modules/scraper/service/suggested-fees-service.ts` or whatever. 
I also see `adapter` subdirectory as the place for putting the code for interacting with external actors: messaging system, database, so this piece code can also be placed at `modules/scraper/adapter/across-serverless-api/[some-service-filename]`